### DISCUSSION TITLE
FIND not working with latest runner

### DISCUSSION THREAD

**Discussion by lionel-faber** [community/community#25039] [Actions]
<p>As a part of my workflow the <code>find</code> command is being used like this:</p>
<pre><code>find "target/release" -maxdepth 1 -type f -exec cp '{}' artifacts /;
</code></pre>
<p>It seemed to work fine for a while but with the latest runner there is an error being returned:</p>
<pre><code>FIND: Parameter format not correct
</code></pre>
<p><a href="https://github.com/maidsafe/safe_client_libs/runs/381063640#step:9:1" rel="nofollow noopener">Working build</a> <br>
<a href="https://github.com/maidsafe/safe_client_libs/runs/399478937#step:9:9" rel="nofollow noopener">Failed build</a> <br>
The runner version has been updated between these two builds. Perhaps that’s the reason?</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/lionel1704">@lionel1704</a> ,</p>
<p>I notice it’s already resolved by adding path for ‘find’ command( ‘/usr/bin/find’).  As you mentioned, this should be caused by the update of Github hosted-runner, bash cannot find the correct ‘find’ command without path specified.</p>
<p>In bash command,  it should use ‘/usr/bin/find’ which is the correct one.</p>
<p>Thanks.</p>

**Comment by thboop** [comment, marked as answer]
<p>There’s a bug for the virtual environments team filed here with more discussion on this issue: <a href="https://github.com/actions/virtual-environments/issues/263" rel="nofollow noopener">https://github.com/actions/virtual-environments/issues/263</a></p>
<p>You can see the workaround we did for the actions/runner here: <a href="https://github.com/actions/runner/pull/285" rel="nofollow noopener">https://github.com/actions/runner/pull/285</a></p>
